.circleci/config.yml

```
@@ -1173,20 +1175,54 @@ jobs:
           spec: next-owner-based-default
           browser: << parameters.browser >>
 
-  integ_react_datastore_custom_pk:
+  integ_react_datastore_custom_pk_unconnected_models:
     parameters:
       browser:
         type: string
     executor: js-test-executor
     <<: *test_env_vars
-    working_directory: ~/amplify-js-samples-staging/samples/react/datastore/v2/custom-pk
+    working_directory: ~/amplify-js-samples-staging/samples/react/datastore/v2/custom-pk-unconnected-models
     steps:
       - prepare_test_env
       - integ_test_js:
-          test_name: 'DataStore - Custom Primary Key'
+          test_name: 'DataStore - Custom Primary Key (Connected Models)'
           framework: react
           category: datastore
-          sample_name: v2/custom-pk
+          sample_name: v2/custom-pk-unconnected-models
+          spec: custom-pk
```

We need to pass in the new spec filenames here and for the other 2. 
```suggestion
          spec: custom-pk-unconnected-models
```